packages/synapse-react-client/src/theme/DefaultTheme.tsx

```diff
@@ -97,7 +97,7 @@ export const defaultMuiThemeOptions: ThemeOptions = {
             fontWeight: 900,
             padding: padding,
             borderRadius: '3px',
-            textTransform: 'capitalize',
+            textTransform: 'var(--synapse-button-text-transform)' as any,
             '&:hover': {
```

Comment:
Thanks, Nick! I haven't tested this, but it looks like exactly what we need to complete https://github.com/bridge2ai/b2ai-standards-registry/issues/249.

While you're at it, could we also fix https://github.com/bridge2ai/b2ai-standards-registry/issues/248 by adding this?

    '&:visited': {
        color: 'inherit',
    },